module/VuFindTheme/src/VuFindTheme/ResourceContainer.php

```diff
@@ -80,25 +80,6 @@ class ResourceContainer
      */
     protected $generator = '';
 
-    /**
-     * Add a Less CSS file.
-     *
-     * @param array|string $less Less CSS file (or array of Less CSS files) to add
-     *
-     * @return void
-     */
-    public function addLessCss($less)
```

Comment:
There is still one remaining reference to this method that needs to be removed from \VuFindTheme\Initializer.